### PR TITLE
CLI: fix -H with watch mode

### LINKED ISSUE #1906: Hierarchy is not preserved when in watch mode
Started with `-w -H` to watch for changes compiles the correct hierarchy on first run but not when compiling the watched file again.

### PR DESCRIPTION
**Pull request by TimothyGu**
Fixes #1906.

Also: do we need a test for this?

### CODE REVIEW

File: bin/jade.js [hunk 1]
```diff
@@ -157,7 +158,7 @@ function watchFile(path, base) {
     if (curr.mtime.getTime() === 0) return;
     // istanbul ignore if
     if (curr.mtime.getTime() === prev.mtime.getTime()) return;
-    tryRender(base || path);
+    tryRender(base || path, rootPath);
```
Comment (yfr) [inline bin/jade.js]: Shouldn't this  also be `render` now?

Comment (TimothyGu) [inline bin/jade.js]: This is in `watchFile`, where `program.watch` is always true and `render === tryRender`, so I didn't bother to change it.

Comment (yfr) [inline bin/jade.js]: Ah. OK fair enough.
 Am 28.03.2015 03:57 schrieb "Timothy Gu" <notifications@github.com>:

**Comment by TimothyGu** [comment]
@ForbesLindesay any comments on this?

**Comment by ForbesLindesay** [comment]
Looks good, but it has merge conflicts.

**Comment by TimothyGu** [comment]
Yeah I'll fix it right now.

**Pull request merged**